lib/proj/projlib.dox

```diff
@@ -36,15 +40,18 @@ versions of GRASS).
 File lib/proj/convert.c contains functions for converting GRASS
 co-ordinate system descriptions to and from the formats used by other
 GIS. Heavy use is made of the OGR library, part of GDAL
-(http://www.gdal.osgeo.org/ogr/).
+(https://gdal.org).
 
 \subsection datum_transformation Datum transformation
 
 GRASS generally supports datum transformation (3 parameters, 7 parameters
 and grid).
 <p>
 
-If a warning appears that a certain datum is not recognised by GRASS and
+Note that datum transformation is handled exclusively by PROJ 6 and later.
+In the case that GRASS is compiled with PROJ 5 or older, the datum
+management is still done within GRASS GIS. Only in that case,
+if a warning appears that a certain datum is not recognised by GRASS and
```

Comment:
I know this is programmer and not user doc, but anyway, the wording seems to be geared towards reader with 10-30 years of GRASS projection handling history in mind instead of just describing the current state. Leaving out things like _Note_, _exclusively_, and _still_, would be probably sufficient fix, but in addition to that, adding a proper condition to the first sentence may be even better, something like: _...is handled exclusively by PROJ if GRASS is compiled with PROJ 6 and later._ Alternatively, the whole version info can be left out from the first sentence because the exception to that rule follows immediately.

Reply:
The important change here is that PROJ 6 introduced a far better handling of datum transformations. Instead of datum X -> WGS84 -> datum Y, PROJ can now do datum X -> datum Y directly. This change was introduced in PROJ with the official release of PROJ 6.0.0 on 1 Mar 2019, less than 2 years ago. The suggested text exactly reflects this transition, and its full implementation in GRASS master, to be backported to GRASS 7.8.

Reply:
and it would make for a good commit message.